src/main/java/uk/gov/register/resources/RecordResource.java

```diff
@@ -97,6 +98,9 @@ public EntryListView getAllEntriesOfARecord(@PathParam("record-key") String key)
     @Timed
     public RecordsView facetedRecords(@PathParam("key") String key, @PathParam("value") String value) {
         List<Record> records = register.max100RecordsFacetedByKeyValue(key, value);
+
+        headerProvider.setAttachmentContentDisposition(key);
+
```

Comment:
can remove whitespace here.

Reply:
I'll do it, but I'm not a big fun of all the lines together, make it more difficult to read it